std/range/package.d

```diff
@@ -2921,13 +2921,13 @@ pure @safe nothrow @nogc unittest
 
 /++
     Convenience function which calls
-    `range.$(REF popFrontN, std, range, primitives)(n) and returns `range`.
+    `range.`$(REF popFrontN, std, range, primitives)`(n)` and returns `range`.
```

Comment:
I this it's better to remove `` `range.` ``, because the link already includes the module name on ddox - see http://dtest.dlang.io/artifact/website-d2e620d6b5bed7f6a50276392a7508505c3c2029-9649de2e244ad8868ecf2a1b2c6ed29c/web/library-prerelease/std/range/drop.html, and more importantly, because it seems to interact badly with ddoc - http://dtest.dlang.io/artifact/website-d2e620d6b5bed7f6a50276392a7508505c3c2029-9649de2e244ad8868ecf2a1b2c6ed29c/web/phobos-prerelease/std_range.html#drop.